timesketch/lib/analyzers/chain_plugins/chrome_download_file.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Plugin for chaining Chrome downloads to filesystem events."""
+
+from timesketch.lib.analyzers.chain_plugins import interface
+from timesketch.lib.analyzers.chain_plugins import manager
+
+
+class ChromeDownloadFilesystemChainPlugin(interface.BaseChainPlugin):
+    """A plugin to chain Chrome downloads to filesystem events."""
+
+    NAME = 'chromefilesystem'
+    DESCRIPTION = (
+        'Plugin to chain Chrome download records to corresponding filesystem '
+        'events and even execution events.')
+
+    SEARCH_QUERY = 'data_type:"chrome:history:file_downloaded"'
+    EVENT_FIELDS = ['full_path']
+
+    def get_chained_events(self, base_event):
+        """Yields an event that is chained or linked to the base event.
+
+        Args:
+            base_event: the base event of the chain, used to construct further
+                queries (instance of Event).
+
+        Yields:
+            An event (instance of Event) object that is linked or chained to
+            the base event, according to the plugin.
+        """
+        target = base_event.source.get('full_path', '')
+        if not target:
+            return
+            yield  # pylint: disable=W0101
+
+        if '\\' in target:
+            separator = '\\'
+        else:
+            separator = '/'
+
+        target = target.split(separator)[-1]
+        search_query = (
```

Comment:
It would be nice to add more things here in the future, like USB events, generic execution events (for all platforms) etc. Can you add a TODO?

Reply:
done